src/main/resources/data/subwild/loot_tables/blocks/molten_andesite.json

```diff
@@ -1,22 +1,43 @@
 {
```

Comment:
Please fix the indentation of this so the git comparison works. If you're editing a file indented with tabs, please make your changes also use tab indentation.

Reply:
Apologies, my changes were made based on vanilla loot tables, it should be resolved now.